tests/server/auth/test_sqlalchemy_store.py

```diff
@@ -424,3 +425,29 @@ def test_delete_registered_model_permission(store):
     ) as exception_context:
         store.get_registered_model_permission(name1, username1)
     assert exception_context.value.error_code == ErrorCode.Name(RESOURCE_DOES_NOT_EXIST)
+
+
+def test_rename_registered_model_permission(store):
+    # create 2 users and create 2 permission for the model registry with the same name
+    model_registry_name = random_str()
+    username1 = random_str()
+    password1 = random_str()
+    _user_maker(store, username1, password1)
+    _rmp_maker(store, model_registry_name, username1, MANAGE.name)
+
+    username2 = random_str()
+    password2 = random_str()
+    _user_maker(store, username2, password2)
+    _rmp_maker(store, model_registry_name, username2, MANAGE.name)
+
+    model_registry_new_name = random_str()
+
+    store.rename_registered_model_permissions(model_registry_name, model_registry_new_name)
+
+    # get permission by model registry new name and all user must have the same new name
+    perm_user_1 = store.get_registered_model_permission(model_registry_new_name, username1)
```

Comment:
Let's check the permission is MANAGE after renaming as well

Reply:
@serena-ruan done and thanks again

Reply:
i added 2 type of permission to check them MANAGE(user1) , READ(user2)